### DISCUSSION TITLE
SDK support for Playwright

### DISCUSSION THREAD

**Discussion by alissa-nonato** [percy/cli#503] [General] [answered]
Hey! Would it be possible to add SDK support for the Playwright end-to-end testing framework? :)

**Comment by Robdel12** [comment, marked as answer]
Hey @alissa-nonato! You can use the Puppeteer SDK with Playwright -- their APIs are exactly the same (err, the APIs we use): https://github.com/percy/percy-puppeteer/discussions/317#discussioncomment-1051888

**Reply by alissa-nonato** [reply]
Thank you!! :)
